docs/_advanced/deferring_initialization.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deferring initialization
+lang: en
+slug: deferring-initialization
+order: 8
+---
+
+<div class="section-content">
+Bolt offers a way to defer full initialization via the `deferInitialization` option and to call the equivalent `App#init()` in your code, putting more control over asynchronous execution required for initialization into your hands as the developer.
```

Comment:
Is the syntax here supposed to be `App#init()` and not `App.init()`? I'm not too familiar with JavaScript.

Reply:
@wongjas `#` here means an instance method in documents - it's not an actual syntax in code. If you say `App.init()`, that sounds like a static method of the class.